static/js/workbox-plugins/cache-first-and-stale-while-revalidate-after-expiration.js

```diff
@@ -10,42 +9,6 @@ const OBJECT_STORE_NAME = 'timestamps';
 //   value but in the background fetch the data from the network to update the cache.
 // eslint-disable-next-line
 class CacheFirstAndStaleWhileRevalidateAfterExpiration {
-    /**
```

Comment:
Turns out these aren't needed since setting them below in the constructor handles the types.